### PR TITLE
Add implementation for FindMeetingQuery

### PR DESCRIPTION
**Pull request by nmsulliv**
Added implementation of FindMeetingQuery which returns possible meeting times.

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -14,10 +14,76 @@
...
+    Collection<TimeRange> possibleMeetingTimes = new ArrayList<>();
+
+    // If there is an invalid duration or no events, return times immediately.
+    long duration = request.getDuration();
+    if (duration > TimeRange.WHOLE_DAY.duration()) {
+      return possibleMeetingTimes;
+    } else if (events.isEmpty()) {
+      possibleMeetingTimes.add(TimeRange.WHOLE_DAY);
+      return possibleMeetingTimes;       
+    } else {
```
Comment (tc466) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:39]: The empty clause is not needed

Comment (nmsulliv) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:39]: Okay, thank you!

Comment (Yunni) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:39]: +1

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 2]
```diff
@@ -14,10 +14,76 @@
...
+    }
+
+    // Reverse the bitset so it only has available minutes.
+    available.flip(0,TimeRange.WHOLE_DAY.duration() + 1);
+
+    // For each of the available times, find the range and add it to the
+    // list of possible meeting times.
+    int rangeLength = 0;
+    for (int i= 0; i < available.length(); i++) {
+      if (available.get(i) == true) {
```
Comment (Yunni) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:74]: nit: `if (available.get(i))`

**Pull request merged**